Contents/mods/Hydrocraft/media/scripts/Camping_HC.txt

```diff
@@ -107,6 +115,7 @@ item HCMagnesiumstriker
 	UseWhileEquipped	=	FALSE,
 	DisplayName		=	Magnesium Striker,
 	Icon			=	HCMagnesiumstriker,
+		DisplayCategory          =   Fuel,
```

Comment:
in vanilla, matches are displayCategory LightSource. Maybe we should use the same for this and lighter